docs/guide/docs/main/memory.md

```diff
@@ -3,57 +3,51 @@ id: memory
 title: Memory
 ---
 
-In the course of a conversation, you may want to ask questions to the user, and remember his answers to use it later.  You may also want to access the values of system parameters such as the values of the slots that were just extracted.
-
+In the course of a conversation, you may want to ask questions to the user, and remember his answers to use it later. You may also want to access the values of system parameters such as the values of the slots that were just extracted.
 
 ## System Parameters
-When a user engages with a bot, Botpress tracks all the variables and parameters associated with that bot as the bot transitions from one state to another.  If you run the emulator you will see the tree of all the system parameters that it tracks.  
 
-You can access these system parameters from the flow builder and from within your code (including in Actions).  To do so, all you need to do to reference a parameter by prefixing the path shown in the emulator with “event.”.
+When a user engages with a bot, Botpress tracks all the variables and parameters associated with that bot as the bot transitions from one state to another. If you run the emulator you will see the tree of all the system parameters that it tracks.
 
-For example, the path shown in the emulator to the language parameter is nlu.language.  You can reference that parameter by adding “event.” to the path shown in the emulator i.e. event.nlu.language.  
+You can access these system parameters from the flow builder and from within your code (including in Actions). To do so, all you need to do to reference a parameter by prefixing the path shown in the emulator with “event.”.
 
-[NLU Language Emulator]
+For example, the path shown in the emulator to the language parameter is nlu.language. You can reference that parameter by adding “event.” to the path shown in the emulator i.e. event.nlu.language.
 
+[NLU Language Emulator]
 
-In the Flow Editor you can access system parameters by bracketing them with two sets of curly brackets.  
+In the Flow Editor you can access system parameters by bracketing them with two sets of curly brackets.
 
 For example in a message you could say:
 
-The bot's language is {{event.nlu.language}}.  
+The bot's language is {{event.nlu.language}}.
```

Comment:
Shouldn't it be the bot detected language instead ? (because if you have a bot in english only, it can still be detected as french or spanish)

Reply:
didn't change this . this is autoformat on save art.